util/EFSOI_Utilities/src/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+cmake_minimum_required(VERSION 2.6)
+if(BUILD_EFSOI)
+
+  file(GLOB LOCAL_SRC ${CMAKE_CURRENT_SOURCE_DIR}/*90)
+
+  message(echo ${PROJECT_BINARY_DIR})
+  
+  set_source_files_properties( ${LOCAL_SRC} PROPERTIES COMPILE_FLAGS ${ENKF_Fortran_FLAGS} )
+  
+  add_executable(global_efsoi.x  ${LOCAL_SRC} )
+  
+  set_target_properties( global_efsoi.x PROPERTIES COMPILE_FLAGS ${ENKF_Fortran_FLAGS} )
+  
+  SET( CMAKE_EXE_LINKER_FLAGS  "${CMAKE_EXE_LINKER_FLAGS} ${OMPFLAG}" )
+  
+# copied from src/enkf/CMakeLists.txt   
+  include_directories(${CMAKE_CURRENT_BINARY_DIR} "${PROJECT_BINARY_DIR}/include/wrf" "${PROJECT_BINARY_DIR}/include/global"  ${CMAKE_CURRENT_BINARY_DIR}/.. ${MPI_Fortran_INCLUDE_DIRS} ${MPI_Fortran_INCLUDE_PATH} ${CORE_INCS} ${NETCDF_INCLUDE_DIRS} ${NCDIAG_INCS} ${FV3GFS_NCIO_INCS}) 
```

Comment:
Why is the `wrf` directory included here?  There is no dependence on wrf in `efsoi` module or main program

Reply:
removed